gcs/detect_gcs_test.go

```diff
@@ -11,20 +11,20 @@ func TestGCSDetector(t *testing.T) {
 	}{
 		{
 			"www.googleapis.com/storage/v1/bucket/foo",
-			"gcs::https://www.googleapis.com/storage/v1/bucket/foo",
+			"https://www.googleapis.com/storage/v1/bucket/foo",
 		},
 		{
 			"www.googleapis.com/storage/v1/bucket/foo/bar",
-			"gcs::https://www.googleapis.com/storage/v1/bucket/foo/bar",
+			"https://www.googleapis.com/storage/v1/bucket/foo/bar",
 		},
 		{
 			"www.googleapis.com/storage/v1/foo/bar.baz",
-			"gcs::https://www.googleapis.com/storage/v1/foo/bar.baz",
+			"https://www.googleapis.com/storage/v1/foo/bar.baz",
```

Comment:
Will a `gcs::` get still work ?

Reply:
Yes. This is the same situation as the FileDetector you asked above. This was for the client.go to know which getter to use.